### ISSUE TITLE
Debug

### ISSUE THREAD

**Issue by matthpich** [uqfoundation/pathos#122]
Hi,
I have a problem running my process in parallel.
"map" keeps crashing with the error:
  "File "/home/xxx/nibr-miniconda2_0.2/lib/python2.7/site-packages/multiprocess/pool.py", line 251, in map
    return self.map_async(func, iterable, chunksize).get()
  File "/home/xxx/nibr-miniconda2_0.2/lib/python2.7/site-packages/multiprocess/pool.py", line 567, in get
    raise self._value
struct.error: 'i' format requires -2147483648 <= number <= 2147483647"
I have not managed to find what goes wrong.
Any hint?
Thanks in advance for your help.
Best,
M

**Comment by mmckerns** [comment]
Post a minimal version of the code that produces the error, as well as the error, and maybe it can be reproduced.  That's the easiest way to get help.

However, whenever I see `raise self._value`, it often points to C-compiler issues... although, the `struct.error` is new to me.  Are you using `pathos`, or `multiprocess` directly?  Post a minimal code sample that reproduces your error.

**Comment by matthpich** [comment]
@mmckerns , thanks for your message.

For your information, I could solve the problem trying to work out a minimal code sample that could reproduce the error.

I am passing `pathos` a `pandas` data frame for parallel row by row computation.
Simple cases with small pandas data frames worked well, as expected, but very large datasets (100k+ rows) led to the error reported above.

**Comment by mmckerns** [comment]
So... you are closing the issue because it's resolved?  I'm not sure I understand.  You say you are still seeing the error, but only for very large datasets. Is that correct?

**Comment by matthpich** [comment]
Yes.
Let me know if I should have left the error open and/or if you are still
interest by a code sample to reproduce the error.

2017-10-10 15:09 GMT-04:00 Mike McKerns <notifications@github.com>:

**Comment by mmckerns** [comment]
If indeed you can make a minimal example, but with a large dataset that causes the error, it's worth interest, and you should post it and leave the ticket open.

**Comment by matthpich** [comment]
Alright.
My apologies for having closed the ticket so prematurely.

2017-10-10 15:19 GMT-04:00 Mike McKerns <notifications@github.com>:

**Comment by episodeyang** [comment]
I am running into this issue as well. The data I'm processing is about 1GB per process.

**Comment by mmckerns** [comment]
@episodeyang: If you can post a minimal example that I can use to confirm the error, either here or in a new issue, that would greatly facilitate any progress.
